releases/release-1.28/release-notes/maps/pr-117211-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 117211
+releasenote:
+  text: |-
+    `kube-apiserver` adds two new metrics `authorization_attempts_total` and `authorization_duration_seconds`
+    that allow users to monitor requests to authorization webhooks, split by result.
```

Comment:
```suggestion
    that allow users to monitor requests to authorization webhooks, split by the results.
```

Reply:
Not sure about this one, in the code the help message says this:
```
Counter of authorization attempts broken down by result. It can be either 'allowed', 'denied', 'no-opinion' or 'error'.
```

I've almost always seen "split by result" used (in e.g. analytics packages), I think it fits well with the itendend meaning: the user will get the requests split by the result, which is a predefined value. Open to debate it further though!